src/main/scala/edu/gemini/gsp/sbtplugin/GspPlugin.scala

```diff
@@ -15,7 +16,8 @@ object GspPlugin extends AutoPlugin {
   object autoImport {
 
     lazy val gspGlobalSettings = Seq(
-      scalaVersion := "2.12.8",
+      scalaVersion := "2.12.9",
+      crossScalaVersions := Seq(scalaVersion.value, "2.13.0"),
```

Comment:
So let's say ocs3 used sbt-gsp. it will be setup to do cross compilation but you'd need to manually do it right?

Reply:
Yes, I think I might just explicitly set `crossScalaVersions` to `Nil` for OCS3, for now.

Reply:
But cross-compiling might be a way to hack on the upgrade, which could be useful. Just don't do the cross build in CI.